src/assets/main.scss

```diff
@@ -121,3 +121,22 @@ $primary: $navbar-background-color;
 .help-image-lg {
   max-height: 60vh;
 }
+
+#contact-message {
+  background-color: $off-white;
+  transform: translateX(-25%);
+  width: 10rem;
+  opacity: 0.8;
+
+  :hover {
+    background-color: $off-white;
+    opacity: 1;
```

Comment:
Might be better if this was `background-color: inherit;` to make it more obvious that the point of this is to ensure styles applied elsewhere are overridden.  Also, the `opacity: 1;` isn't doing anything, because the opacity is being set directly on the element in js, which has a higher specificity, so it can just be removed, I think.